vendor/brave-ios/Ledger/BATBraveLedger.mm

```diff
@@ -965,6 +965,37 @@ - (void)clearState:(const std::string&)name
   [self savePrefs];
 }
 
+- (bool)getBooleanOption:(const std::string&)name
+{
+  // TODO implement
```

Comment:
Is there going to be globally defined strings for checking? Future PR?

Reply:
yes in new PR I will add examples of usage, but every platform can implement their own version of it

Reply:
on desktop we will have `map<string, type>` and simple `at` to get it

Reply:
https://github.com/brave/brave-core/pull/3628 client implementation